src/Microsoft.AspNet.Authentication/AuthenticationOptions.cs

```diff
@@ -27,6 +27,13 @@ public string AuthenticationScheme
         }
 
         /// <summary>
+        /// If true the authentication middleware alter the request user coming in and
```

Comment:
Can we rephrase this a little bit something like.. authentication middleware `will` set the request's <see cref=Request.User> and handle 401 unauthorized responses.